tests/plan/lint/main.fmf

```diff
@@ -1,2 +1,7 @@
 summary: Checks whether plan linting works correctly
-test: ./test.sh
+
+/execute:
+    test: ./execute.sh
+
+/discover:
+    test: ./discover.sh
```

Comment:
For now I'd suggest to use a single test to cover the `tmt plan lint` functionality and keep individual step coverage as `beakerlib` phases. If the coverage substantially grows we can split later.

Reply:
ok

Reply:
merged to one file.